drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -196,6 +203,7 @@ function DragNDropTemplates(url_name) {
                         'data-zone_id': zone.id,
                         'data-zone_align': zone.align,
                         'role': 'button',
+                        'aria-describedby': getItemPropertyFromSet(items_in_zone_descriptions, "displayName").join(", "),
```

Comment:
How about "Items placed: " or "Items placed here: "

Also you are only using `getItemPropertyFromSet` in this one place, so you may as well just write `items_in_zone.map(function (item) { return item.displayName; })`